integration-tests/go.mod

```diff
@@ -8,133 +8,88 @@ require (
 	github.com/onsi/gomega v1.19.0
 	github.com/rs/zerolog v1.27.0
 	github.com/satori/go.uuid v1.2.0
-	github.com/smartcontractkit/chainlink-testing-framework v1.2.9-0.20220623154152-6ff785a4db4e
-	github.com/smartcontractkit/helmenv v1.2.4
+	github.com/smartcontractkit/chainlink-env v0.2.13
+	github.com/smartcontractkit/chainlink-testing-framework v1.3.11
+)
+
+require (
+	github.com/aws/constructs-go/constructs/v10 v10.1.22 // indirect
+	github.com/aws/jsii-runtime-go v1.59.0 // indirect
+	github.com/cdk8s-team/cdk8s-core-go/cdk8s/v2 v2.3.14 // indirect
+	github.com/fatih/structs v1.1.0 // indirect
```

Comment:
lol Why do go mod do this? :thinking: 
Let's combine these separate `// indirect` blocks and then tidy again.